about/jobs/template.md

```diff
@@ -0,0 +1,81 @@
+# Template for your LinkedIn
+
+We want to be promote ourselves and FAC as _much_ as we can on our LinkedIn profiles and encourage people to contact us whether it be to source candidates for vacancies or freelance work.
+
+This way, we'll _all_ have _many_ choices for what to do when we're in a position to go into a full-time job or work on freelance projects.
```

Comment:
Worth mentioning here that you're making an assumption with the suggestions below that the person wants to work in technology, most likely as a web developer and they should be adjusted accordingly